package.json

```diff
@@ -134,7 +138,8 @@
     ],
     "moduleNameMapper": {
       "\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$": "<rootDir>/src/test/fixtures/mocks/file-mock.js",
-      "\\.(css|less)$": "<rootDir>/src/test/fixtures/mocks/style-mock.js"
+      "\\.(css|less)$": "<rootDir>/src/test/fixtures/mocks/style-mock.js",
+      "^gecko-profiler-demangle$": "<rootDir>/src/test/gecko-profiler-demangle-mock.js"
```

Comment:
I believe this `moduleNameMapper` is not needed, as the file can be placed in `/__mocks__` and it will be picked up automatically using the mocking resolution scheme that jest uses, where it will find files in the nearest `__mocks__` directory. It will use the root `__mocks__` directory for node modules. See `__mocks__/copy-to-clipboard.js` for an example.